presto-main/src/test/java/com/facebook/presto/type/TestMapOperators.java

```diff
@@ -442,22 +442,35 @@ public void testJsonToMap()
     public void testElementAt()
             throws Exception
     {
+        // empty map
```

Comment:
The original tests didn't cover reasonable corner cases. It's also not grouped in a way that helps the reader tell what's covered.